key_management/src/wallet.rs

```diff
@@ -176,6 +176,21 @@ pub fn find_key<T: KeyStore>(addr: &Address, keystore: &T) -> Result<Key, Error>
     Ok(new_key)
 }
 
+pub fn try_find<T: KeyStore>(addr: &Address, keystore: &mut T) -> Result<KeyInfo, Error> {
+    let key_string = format!("wallet-{}", addr.to_string());
+    match keystore.get(&key_string) {
+        Ok(k) => Ok(k),
+        Err(_) => {
+            let mut new_addr = addr.to_string();
+            new_addr.replace_range(0..1, "t");
```

Comment:
Can you add some sort of todo to replace the network prefix default in address, or change this with it? Whenever that is changed, this would be a very tricky workaround to debug?